client/src/pages/ForumPage/ForumPage.js

```diff
@@ -19,8 +22,32 @@ function pillClick(val) {
 const handleClick = e => {
   console.log("Clicked");
 };
+class ForumPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      topics: [],
+      hashtags: [],
+    }
+  }
+
+componentDidMount() {
+  axios.get(`${BASE_URL}/hashtags`)
+    .then(response => {
+      this.setState({ 
+        hashtags: response.data
+      })
+      console.log(response.data);
```

Comment:
Can you remove this and any other `console.log`s?

Reply:
Removed console.logs from the page